charts/linkerd-control-plane/values.yaml

```
@@ -286,10 +289,16 @@ proxy:
     collector:
       # -- The collector endpoint to send traces to.
       endpoint: ""
-      # -- The identity of the collector in the linkerd mesh. If the collector
-      # is unmeshed, this should remain unset.
+      # -- The identity of the collector in the linkerd mesh.
       meshIdentity:
+        # -- Service account name for the trace collector. If there's no explicitly set service account for the trace
+        # collector, this should be set to the name of the deployment/statefulset/daemonset of the trace collector
+        # instead.
         serviceAccountName: ""
+        # -- Service account namespace for the trace collector. If there's no explicitly set service account for the
+        # trace collector, this should be set to the namespace of the deployment/statefulset/daemonset of the trace
+        # collector instead.
+        serviceAccountNamespace: ""
```

```suggestion
        namespace: ""
```

blegh, sorry to yakshave this, but we are probably better off just calling this 'namespace'

---

Sorry, I think I'm partial to `serviceAccountName` and `namespace` here:

* these are how these fields are named on pod spec/metadata resources, so it's consistent
* the 'name' is ambiguous. i think it's clearer if we document that this is explicitly a serviceAccountName (as on the pod spec).
* the namespace is more general -- it includes the workloads and the services blah blah blah